main.js

```diff
@@ -0,0 +1,379 @@
+$(document).ready(loadDocument);
+
+//########################################## GLOBAL VARIABLES ###################################
+// Player Info
+var player1 = null;
+var player2 = null;
+
+var currentPlayer = player1;
+var currentPlayerStatus = true; // true is player 1 and false is player 2
+var screenClickable = true;
+
+
+
+var winCount = 4;//for connect 4 or even higher
+
+var gameBoardArray = [
+    ['','','','','',''],
+    ['','','','','',''],
+    ['','','','','',''],
+    ['','','','','',''],
+    ['','','','','',''],
+    ['','','','','',''],
+    ['','','','','','']
+]; //gameBoardArray template
+
+
+function loadDocument(){
+    addClickHandlers();
+    
+}
+
+function addClickHandlers(){
+    $(".column").click(columnClicked);
+    $(".playerColumn img").click(powerupButtonClicked); // sets image as powerUp Button
+    $(".resetGame").click(resetGame);
+    $(".playerCharacterSelectionModal img").click(characterClicked);
+    $(".gameStart").click(gameStart);
+}
+
+function gameStart() {
+    $(".gameStart").addClass('hiddenElement'); 
+    $(".playerCharacterSelectionModal").removeClass('hiddenElement');
+    bgMusicPlay();
+}
+
+
+function columnClicked(){
+    if(!screenClickable){//if screenClickable = true then you can click.
+        return;
+    }
+
+    var columnClicked = $(this).attr("column");
+
+    columnClicked = parseInt(columnClicked);
+    //drop token and update game board. //if pop up modal for sorry try again.
+    var currentRowDroppedIn = dropTokenCol( currentPlayer, columnClicked);
+    if (currentRowDroppedIn === null){
+        return;
+    }
+    //check to see if powerup is allowed OR if player has won.
+    tokenPlacementCheck( currentPlayer, columnClicked, currentRowDroppedIn );
+}
+
+function powerupButtonClicked(){
+    //activates powerUp if currentplayer powerupheld is = true;
+    var test = $(this).hasClass(currentPlayer.characterType.characterToken);
+    if(currentPlayer.powerupHeld && test) {
+        usePowerup(currentPlayer.characterType.characterPowerup);
+    
+        currentPlayer.powerupHeld = false;
+        if(currentPlayerStatus) {
+            $('#player1 img').removeClass('animatePowerupButton');
+        }else{
+            $('#player2 img').removeClass('animatePowerupButton');
+        }
+        updateDOM();
+    }
+}
+
+function characterClicked() {
+    if($(this).hasClass("character")) {
```

Comment:
do you remove the class after it has been selected?  Do you apply the click handler to buttons other than the character select buttons?  Why have this check?